consul/consul-k8s/consul.sysconfig

```diff
@@ -0,0 +1,3 @@
+CONFIG_DIR=/etc/consul
```

Comment:
Oh, I should've said "in a separate file"? Why not have it defined in the systemd unit directly?